src/main/java/mServer/crawler/sender/arte/MediathekArte_de.java

```diff
@@ -64,210 +63,193 @@
      * 2. für jede Kategorie die Unterkategorien ermitteln: URL_CATEGORY
      * 3. für jede Unterkategorie die enthaltenen ProgramId ermitteln: URL_SUBCATEGORY
      * 4. für alle ProgramIds die Videoinformationen laden (wie kurze Variante)
-     */
-    private static final Logger LOG = LogManager.getLogger(MediathekArte_de.class);
-    private final static String SENDERNAME = Const.ARTE_DE;
-    private static final String ARTE_API_TAG_URL_PATTERN = "https://api.arte.tv/api/opa/v3/videos?channel=%s&arteSchedulingDay=%s";
-    
-    private static final String URL_STATIC_CONTENT = "https://static-cdn.arte.tv/components/src/header/assets/locales/%s.json?ver=%s";
-    private static final String URL_CATEGORY = "https://www.arte.tv/guide/api/api/pages/category/%s/web/%s";
-    private static final String URL_SUBCATEGORY = "https://www.arte.tv/guide/api/api/videos/%s/subcategory/%s/?page=%s";
-    private static final String VERSION_STATIC_CONTENT = "2.3.13";
-
-    private static final DateTimeFormatter ARTE_API_DATEFORMATTER = DateTimeFormatter.ofPattern("yyyy-MM-dd");
-    protected String LANG_CODE = "de";
-    protected String URL_CONCERT = "http://concert.arte.tv/de/videos/all";
-    protected String URL_CONCERT_NOT_CONTAIN = "-STF";
-    protected String TIME_1 = "<li>Sendetermine:</li>";
-    protected String TIME_2 = "um";
-
-    public MediathekArte_de(FilmeSuchen ssearch, int startPrio) {
-        super(ssearch, SENDERNAME,/* threads */ 2, /* urlWarten */ 200, startPrio);
+   */
+  private static final Logger LOG = LogManager.getLogger(MediathekArte_de.class);
+  private final static String SENDERNAME = Const.ARTE_DE;
+  private static final String ARTE_API_TAG_URL_PATTERN = "https://api.arte.tv/api/opa/v3/videos?channel=%s&arteSchedulingDay=%s";
+
+  private static final String URL_SUBCATEGORY
+          = "https://www.arte.tv/guide/api/api/zones/%s/web/videos_subcategory_%s/?page=%s&limit=100";
+
+  private static final String[] SUBCATEGORIES = new String[]{
+    "AJO", "AUV", "KUL", "DCY", "ENQ", "JUN",
+    "ACC", "CMG", "FLM", "CMU", "MCL",
+    "CHU", "FIC", "SES",
+    "ART", "POP", "IDE",
+    "ADS", "CLA", "JAZ", "MUA", "MUD", "OPE",
+    "ENB", "ENN", "SAN", "TEC",
+    "ATA", "EVA", "NEA", "VIA",
+    "CIV", "LGP", "XXE"
+  };
+
+  private static final DateTimeFormatter ARTE_API_DATEFORMATTER = DateTimeFormatter.ofPattern("yyyy-MM-dd");
+  protected String LANG_CODE = "de";
+  protected String URL_CONCERT = "http://concert.arte.tv/de/videos/all";
+  protected String URL_CONCERT_NOT_CONTAIN = "-STF";
+  protected String TIME_1 = "<li>Sendetermine:</li>";
+  protected String TIME_2 = "um";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "TIME_2" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00116)